docs/src/testing/advanced/fuzz-testing.md

```diff
@@ -0,0 +1,59 @@
+# Fuzz Testing
+
+In many cases, a test needs to verify function behavior for multiple possible values.
+While it is possible to come up with these cases on your own, it is often impractical, especially when you want to test
+for a large number of possible arguments.
```

Comment:
not sure if `test for` or `test against` is correct

Reply:
I think the second one sounds, better. Changing